sycl/test/graph/graph-explicit-repeated-exec.cpp

```diff
@@ -0,0 +1,67 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+#include <CL/sycl.hpp>
```

Comment:
```suggestion
#include <sycl/sycl.hpp>
```
This is the preferred include for code written against SYCL 2020.

Reply:
Thanks, I changed that.